allocator.py

```diff
@@ -0,0 +1,92 @@
+from people import Fellow, Staff
+from room import Office, LivingSpace
+import random
+
+class Dojo(object):
+    def __init__(self):
+        self.room_name = []
+        self.offices = {}
+        self.livings = {}
+
+        self.staff_list = []
+        self.fellow_list = []
+
+    def create_room(self, is_office, room_type):
+        rooms = room_type
+        # check if its staff or fellow and create person by calling staff and fellow objects
+
+        if is_office:
+            for room in rooms:
+                create_office = Office(room)
+                self.offices[create_office.name] = []
+                print("Office ", room, " has been created")
+
+        else:
+            for room in rooms:
+                create_living = LivingSpace(room)
+                self.livings[create_living.name] = []
+                print("Living space ", room, " has been created")
+
+
+    def create_person(self, fname, lname, is_fellow, need_room='No'):
+        names = fname + " " + lname
+        if is_fellow:
+            create_fellow = Fellow(names)
+            self.fellow_list.append(create_fellow)
+
+            if need_room == 'No':
+                need_room = False
+                self.assign_room(create_fellow, need_room)
```

Comment:
Using `create_fellow` as the variable name holding the fellow that has been created on line 34 makes sense, but when `create_fellow` is being passed to `assign_room()` it sounds very confusing because it seem like you are trying to create a fellow when assigning a room, though that isn't the case. Consider renaming your variable to something that will be more appropriate throughout its use (something like `person` or `fellow`)